### PR TITLE
MOBILE-7022/32: [iOS][Swift] Migrate OXARewardedAdUnit/OXAInterstitialAdUnit

### PR DESCRIPTION
**Pull request by VadimKhohlovOpenX**
** JIRA issue **
https://openxtechinc.atlassian.net/browse/MOBILE-7022
https://openxtechinc.atlassian.net/browse/MOBILE-7032

**Description of change**

Rewritten OXARewardedAdUnit/OXAInterstitialAdUnit on Swift
PrebidServerUITests swift 2021-05-25 10-44-36

### CODE REVIEW

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    @objc public override init(configId: String, minSizePercentage: CGSize) {
+        super.init(
+            configId: configId,
+            minSizePercentage: minSizePercentage,
+            eventHandler: PBMInterstitialEventHandlerStandalone())
+    }
+
+    @objc public override init(configId: String, minSizePercentage:CGSize, eventHandler: Any) {
+        super.init(configId: configId, minSizePercentage: minSizePercentage, eventHandler: eventHandler)
```
Comment (YuriyVelichkoOpenX) [inline PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift]: Let's write similar things in similar way. In the above method the parameters of super.init are placed on own lines, in this case all of them on the single line.

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    
+    override init(configId:String, minSizePerc: NSValue?, eventHandler:Any?) {
+        super.init(configId: configId, minSizePerc:minSizePerc, eventHandler:eventHandler)
+    }
+    
+    
+    // MARK: - Protected overrides
+
+    @objc public override func callDelegate_didReceiveAd() {
+        let delegate = self.delegate as? PBMInterstitialAdUnitDelegate
```
Comment (YuriyVelichkoOpenX) [inline PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift]: this code construction requires let or guard. The current one looks strange though it makes sense.

**Review by YuriyVelichkoOpenX — APPROVED** [review]
small Swift style fixes are required

**Review by YuriyVelichkoOpenX — CHANGES_REQUESTED** [review]
Some Swift-style changes are required

**Comment by VadimKhohlovOpenX** [comment]
Done

**Review by YuriyVelichkoOpenX — APPROVED** [review]
LGTM

**Pull request merged**